REST_API_DOCS.md

```
@@ -0,0 +1,1861 @@
+# 클라우드 커뮤니티 REST API Docs
+
+### 내가 좋아요한 게시글 목록
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)